src/Accounts.js

```diff
@@ -0,0 +1,517 @@
+/* eslint-disable max-len */
+
+import { each, keys, includes, trim, isEmpty, merge, extend } from 'lodash';
+import { Iterable } from 'immutable';
```

Comment:
Yeah I could and would prefer to. Not using immutable was an effort to stay vanilla because I wasn't sure of what consumption of this would be like. It's possible to use an immutable store with Accounts already, I'll make the additional changes to use immutable.